ci/test/00_setup_env_mac.sh

```diff
@@ -15,3 +15,4 @@ export RUN_UNIT_TESTS=false
 export RUN_FUNCTIONAL_TESTS=false
 export GOAL="all deploy"
 export BITCOIN_CONFIG="--with-gui --enable-reduce-exports --disable-miner --with-boost-process"
+export NO_WERROR=1
```

Comment:
wait, is it disabled for clang too? does it mean no -werror enabled at all on CI?

Reply:
It is disabled in _this_ PR but re-enabled in [dash#6638](https://github.com/dashpay/dash/pull/6638) for Clang and [dash#6639](https://github.com/dashpay/dash/pull/6639) for GCC, please refer to PR description.